portfolio/src/main/webapp/script.js

```diff
@@ -48,24 +48,44 @@ function constructImage(imageIndex){
   txtContainer.appendChild(txtElement);
 } 
 
+/*
+ * Extracts max comment from input, otherwise returns 5 
+ */
+function getMaxComment() {
+  numComments = document.getElementById('maxComments').value ;
+  return numComments || 5; 
+}
+
 /**
- * Fetches array list from the \data server and adds them to index.html
+ * Fetches comments from the \data server and adds them to index.html
  */
 function getComment() {
```

Comment:
nit: this should be named getComments (plural) since you'll be grabbing multiple comments.